Appium/Dockerfile

```diff
@@ -98,6 +100,8 @@ RUN curl -sL https://deb.nodesource.com/setup_12.x | bash && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
     apt-get clean
 
+RUN appium driver install uiautomator2 && appium driver install xcuitest
```

Comment:
@jlipps can we install multiple drivers within single driver install` call?

Reply:
no, but you can install multiple drivers at the same time as installing appium, i think it's `npm install -g appium@next --drivers=uiautomator2,xcuitest`

Reply:
sadly, https://github.com/appium/appium/issues/16924